fuzz/src/main/java/edu/berkeley/cs/jqf/fuzz/ei/ZestGuidance.java

```diff
@@ -810,6 +813,10 @@ protected void saveCurrentInput(Set<Object> responsibilities, String why) throws
         if (SAVE_ONLY_VALID == false || currentInput.valid) {
             writeCurrentInputToFile(saveFile);
             infoLog("Saved - %s %s %s", saveFile.getPath(), how, why);
+            if (EXACT_CRASH_PATH != null) {
+                File exactCrashFile = new File(EXACT_CRASH_PATH);
+                writeCurrentInputToFile(exactCrashFile);
+            }
```

Comment:
If the property is EXACT_CRASH_PATH, then shouldn't this logic be next to the saving of failures instead of saving every single input? Maybe it should be near the following: https://github.com/rohanpadhye/jqf/blob/d2c2b3f60491a4a5812cb1d20daa6542063abdca/fuzz/src/main/java/edu/berkeley/cs/jqf/fuzz/ei/ZestGuidance.java#L704-L709

Reply:
you are right. I mixed up. pushing a fix for both now.